### PR TITLE
access cosmology specified in LensModel in magnification_finite_adaptive

### PR DESCRIPTION
**Pull request by dangilman**
I just added that attempts to access the cosmo attribute of _lensModel, and only reverts to the default astropy cosmology if the _lensModel class does not have a .cosmo attribute for whatever reason

### CODE REVIEW

File: lenstronomy/LensModel/lens_model_extensions.py [hunk 1]
```diff
@@ -72,8 +72,12 @@ def magnification_finite_adaptive(self, x_image, y_image, source_x, source_y, kw
         """
 
         if cosmo is None:
-            from astropy.cosmology import default_cosmology
-            cosmo = default_cosmology.get()
+            # try to access a cosmology specified in the lens model first
+            try:
```
Comment (sibirrer) [inline lenstronomy/LensModel/lens_model_extensions.py:76]: you actually don't need the try and error. If you look at the LensModel class, this instance is always generated

**Review by sibirrer — CHANGES_REQUESTED** [review]
minor change to avoid try and error when not necessary (also avoids further testing/test coverage decrease in this PR)

**Comment by dangilman** [comment]
coverage went down only because I removed some lines that were tested

**Comment by sibirrer** [comment]
Thanks! Test coverage decrease due to fewer lines of code

**Pull request merged**